src/applications/facility-locator/components/ResultsList.jsx

```diff
@@ -81,19 +81,70 @@ class ResultsList extends Component {
       );
     }
 
-    if (error && error.find(err => err.code === timeoutErrorCode)) {
-      /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
-      return (
-        <div
-          className="search-result-title facility-result"
-          ref={this.searchResultTitle}
-        >
-          The search has timed out, please retry your search.
-        </div>
-      );
-      /* eslint-enable jsx-a11y/no-noninteractive-tabindex */
+    if (error) {
+      // For some reason, an error can be an HTTP response, or just a string.
+      if (error.find) {
+        const timedOut = error.find(err => TIMEOUTS.has(err.code));
+        if (timedOut) {
+          return (
+            <div
+              className="search-result-title facility-result"
+              ref={this.searchResultTitle}
+            >
+              <p>We’re sorry. We couldn’t complete your request.</p>
+              <strong>To try again, please:</strong>
+              <ul className="vads-u-margin-y--1p5">
+                <li>
+                  <strong>Add a service type</strong> (like “primary care”), and
+                  select the option that best meets your needs. This will help
+                  to narrow your search.
+                </li>
+                <li>
+                  <strong>Or enter a different search term</strong> (street,
+                  city, state, or postal code).
+                </li>
+              </ul>
+              Then click <strong>Search</strong>.
+            </div>
+          );
+        }
+      } else {
+        return (
+          <div
+            className="search-result-title facility-result"
+            ref={this.searchResultTitle}
+          >
+            We’re sorry. We couldn’t complete your request. Please refresh the
+            page or try again later.
+          </div>
+        );
+      }
```

Comment:
I guess we didn't have a standard catch-all for error-handling, so adding that here.